src/status.js

```diff
@@ -33,4 +33,13 @@ export function clearCompleted() {
   });
   localStorage.setItem('todo', JSON.stringify(newTodos));
   window.location.reload();
-}
\ No newline at end of file
+}
+
+// export function clearCompleted() {
+//   const newTodos = todos.filter((todo) => todo.completed !== true);
+//   newTodos.forEach((object, index) => {
+//     object.index = index + 1;
+//   });
+//   localStorage.setItem('todo', JSON.stringify(newTodos));
+//   window.location.reload();
+// }
```

Comment:
Great Job @1GORDON 

- [ ] Kindly remove commented code from your project as this is not regarded as a good practice